node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
```

Comment:
Good! You actually don't need to have a separate function here. You could just do all the statements one after the other.